dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java

```diff
@@ -2112,6 +2114,47 @@ private void refreshWorkFlowActionletMap() {
 		}
 	}
 
+	private void scanDefaultPackagesForActionlets() {
+
+		final String [] packagesForScan = Config.getStringArrayProperty(WebKeys.WORKFLOW_BASE_PACKAGES_ACTIONLET_CLASSES,
+				"com.dotmarketing.portlets.workflows.actionlet.custom");
```

Comment:
Shouldn't we use `dotcms` instead of `dotmarketing`?

Reply:
yeah but it is a legacy package where we have the actionlets, what do you think @wezell